.github/workflows/repo2cpe-download.yaml

```diff
@@ -3,6 +3,9 @@ name: Download and Upload Repository Mappings
 on:
   schedule:
     - cron: "0 0 * * *"
+  push:
+    branches:
+      - "yli3/*"
```

Comment:
Nit: Maybe if you hard-code `scanner/*` here instead of `yili3/*` we would be able to trigger this without having to have this changed in every PR you create.